lib/UnoCore/Targets/Android/Android.uxl

```diff
@@ -79,6 +79,9 @@
     <Set Outputs.AAR="app/build/outputs/aar/app-@(Build.Configuration:ToLower).aar" />
     <Set Outputs.APK="app/build/outputs/apk/@(Build.Configuration:ToLower)/app-@(Build.Configuration:ToLower).apk" />
     <Set Outputs.Bundle="app/build/outputs/bundle/@(Build.Configuration:ToLower)/app-@(Build.Configuration:ToLower).aab" />
+    
+    <Set ProGuard.PrependRulesFile="@(Project.Android.Optimize.PrependRulesFile:ToLower)" />
```

Comment:
Naming: I think `Proguard` should be written with lower-case `g` (several instances). (In Gradle we write `proguardSomething` and not `proGuardSomething`.)